source/jormungandr/jormungandr/scenarios/distributed.py

```diff
@@ -46,33 +50,45 @@ class PartialResponseContext(object):
     orig_fallback_durations_pool = None
     dest_fallback_durations_pool = None
     partial_response_is_empty = True
+    orig_places_free_access = None
+    dest_places_free_access = None
+    journeys_to_modes = dict()  # Map of journeys's internal id to fallback modes.
 
 
-class Scenario(new_default.Scenario):
-    def __init__(self):
-        super(Scenario, self).__init__()
+class Distributed:
```

Comment:
```suggestion
class Distributed(object):
```
https://wiki.python.org/moin/NewClassVsClassicClass